batcher/batcher.mligo

```diff
@@ -40,6 +40,11 @@
 [@inline] let cannot_update_scale_factor_to_more_than_the_maximum : nat          = 136n
 [@inline] let cannot_remove_swap_pair_that_is_not_disabled : nat                 = 137n
 [@inline] let token_name_not_in_list_of_valid_tokens : nat                       = 138n
+[@inline] let no_orders_for_user_address : nat                                   = 139n
+[@inline] let cannot_cancel_orders_for_a_batch_that_isn_not_open : nat           = 140n
```

Comment:
typo btw

Reply:
This is from the cancellations branch. I have fixed there